org.freedesktop.Sdk.Extension.node10.yaml

```
@@ -0,0 +1,35 @@
+app-id: org.freedesktop.Sdk.Extension.node10
+branch: '18.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /usr/lib/sdk/node10
+modules:
+  - name: node
+    sources:
+      - type: archive
+        url: https://nodejs.org/dist/v10.15.3/node-v10.15.3.tar.gz
+        sha256: db460a63d057ac015b75bb6a879fcbe2fefaaf22afa4b6f6445b9db61ce2270d
+
+  - name: scripts
+    buildsystem: simple
+    build-commands:
+      - 'cp enable.sh install.sh /usr/lib/sdk/node10'
+      - 'mkdir -p /usr/lib/sdk/node10/share/metainfo'
+      - 'cp org.freedesktop.Sdk.Extension.node10.appdata.xml /usr/lib/sdk/node10/share/metainfo'
+      - 'appstream-compose --basename=org.freedesktop.Sdk.Extension.node10 --prefix=${FLATPAK_DEST} --origin=flatpak org.freedesktop.Sdk.Extension.node10'
+    sources:
+      - type: script
+        dest-filename: install.sh
+        commands:
+          - 'cp -r /usr/lib/sdk/node10 ${FLATPAK_DEST}/node'
+      - type: script
+        commands:
+          - 'export PATH=$PATH:/usr/lib/sdk/node10/bin'
```

Pointing `npm_config_nodedir` to the extension point may be helpful, at leat to prevent node-gyp from downloading node headers.
```suggestion
          - 'export PATH=$PATH:/usr/lib/sdk/node10/bin'
          - 'export npm_config_nodedir=/usr/lib/sdk/node10'
```